recipes/scikit-image/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.12.3" %}
+
+package:
+  name: scikit-image
+  version: {{ version }}
+
+source:
+  fn: scikit-image-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
+  md5: 04ea833383e0b6ad5f65da21292c25e1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - cython
+    - python
+    - six
+    - numpy x.x
+    - scipy
+  run:
+    - python
+    - six
+    - numpy x.x
+    - scipy
+    - matplotlib
+    - networkx
+    - pillow
+
+test:
+  imports:
+    - skimage
+
+about:
+  home: http://scikit-image.org/
+  license: BSD 3-Clause
+  summary: Image processing routines for SciPy
+
+extra:
+  recipe-maintainers:
+    - ivoflipse
+    - jakirkham
+    - Korijn
+    - msarahan
```

Comment:
Added @ivoflipse and @msarahan per request. Was suggested by @ivoflipse to add @Korijn too. Is this ok @Korijn?